src/EffectTransform.js

```diff
@@ -187,11 +190,13 @@ class EffectTransform {
      * @return {twgl.v3} dst - The coordinate after being transform by effects.
      */
     static transformPoint (drawable, vec, dst = twgl.v3.create()) {
+        const effects = drawable.getEnabledEffects();
+        // If no effects are enabled, don't get anything, create anything, or set anything; just give back the input.
+        if (effects === 0) return vec;
```

Comment:
Same here. Keep the early exit but move it to after `dst` is initialized and return `dst`.